examples/src/agents/stagehand_ai_web_browser.ts

```diff
@@ -0,0 +1,42 @@
+import { StagehandToolkit } from "@langchain/community/agents/toolkits/stagehand";
```

Comment:
Just import it into your `.mdx` file so that you get type checking, linting etc.

Reply:
added `StagehandToolkit` to my `.mdx` for type checking and linting.

Reply:
note that for this I had to remove this dependency from the stagehand lib itself. It will only work with stagehand version 1.2.0+